src/assets/i18n/it.json

```diff
@@ -71,7 +71,7 @@
         }
       },
       "ui": {
-        "newFilter": "New filer"
+        "newFilter": "New filter"
```

Comment:
```suggestion
        "newFilter": "Nuovo filtro"
```

Reply:
Same as french, italian will come later (if we find some one speaking italian for this task, since I don't speak italian)